### PR TITLE
Fix setWindow bug, add CS pin for Clue compatibility.

### PR DESCRIPTION
**Pull request by jmstriegel**
- corrected bit shift in setWindow that broke high addresses
- added csPin to constructor (will now work with adafruit clue)
- small adjustments and comments to init routine based on working arduino driver

### CODE REVIEW

File: st7789/st7789.go [hunk 1]
```diff
@@ -224,21 +253,22 @@ func (d *Device) SetRotation(rotation Rotation) {
 	madctl := uint8(0)
 	switch rotation % 4 {
 	case 0:
-		madctl = MADCTL_MX | MADCTL_MY
+		madctl = MADCTL_MX | MADCTL_MY | MADCTL_RGB
```
Comment (conejoninja) [inline st7789/st7789.go:256]: are you sure MADCTL_RGB (0x00) is required here?

Comment (jmstriegel) [reply st7789/st7789.go:256]: Yeah, not required at all. It doesn't affect the data byte, but I thought it was more clear that MADCTL has an effect on RGB/BGR memory ordering in addition to rotation.

Comment (jmstriegel) [reply st7789/st7789.go]: Latest commit removes this change.

**Comment by jmstriegel** [comment]
Hi @conejoninja - checking to see if this patch needed anything else to merge. I'm making good progress on a further update that adds basic support for vsync timing. I'm new here, so please let me know if there's anything I should be doing differently when contributing updates.

**Comment by conejoninja** [comment]
Merged! Great work, thanks for your contribution. 

Just one small detail, for future PR, please do them against `dev` branch, it's better if you use it as a base branch too for your changes.

**Comment by jmstriegel** [comment]
Thanks! Will do.

**Pull request merged**